### PR TITLE
Refactorings with refurb pre-commit hook

### PR DESCRIPTION
**Pull request by danyeaw**
This PR makes fixes suggested by the refurb pre-commit hook and adds the hook.

### PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I have read, and I am following the [Contributor guide](https://github.com/gaphor/gaphor/blob/main/CONTRIBUTING.md)
- [X] I have read, and I understand the GNOME [Code of Conduct](https://wiki.gnome.org/Foundation/CodeOfConduct)

### PR Type
What kind of change does this PR introduce?

- [ ] Bug fix
- [ ] Feature
- [X] Chore (refactoring, formatting, local variables, other cleanup)
- [ ] Documentation content changes

### What is the current behavior?

Issue Number: N/A

### Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

**Review by amolenaar — APPROVED** [review]
Looks good.

I think it needs some user testing in places where files are involved (loading, saving, custom `console.py` script.

I have one suggestion in the SysML label mapping.

File: gaphor/SysML/diagramlabel.py [hunk 1]
```diff
@@ -20,14 +20,11 @@ def element_type(el):
             return "package"
         if isinstance(el, Requirement):
             return "requirement"
-        if isinstance(el, StateMachine):
-            return "state machine"
-        return ""
+        return "state machine" if isinstance(el, StateMachine) else ""
```
[the remark this answers is not in the document]
Comment (danyeaw) [reply gaphor/SysML/diagramlabel.py:23]: Good catch @amolenaar, I updated it :+1:

File: gaphor/application.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def active_session(self):
 
     def new_session(self, *, filename=None, template=None, services=None, force=False):
         filename = Path(filename) if filename else None
-        if filename is None and template is None:
+        if filename is None is template:
```
[the remark this answers is not in the document]
Comment (danyeaw) [reply gaphor/application.py:91]: ya, wild right?

**Comment by danyeaw** [comment]
@amolenaar Any thoughts on how we perform user testing on this?

**Comment by amolenaar** [comment]
It seems to work fine.

File: gaphor/SysML/diagramlabel.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (amolenaar) [reply gaphor/SysML/diagramlabel.py:23]: Forgot to push?

**Pull request merged**